syft/file/license.go

```diff
@@ -0,0 +1,33 @@
+package file
```

Comment:
nit: should all of this be added in the follow up PR instead?

Reply:
Yea I can remove this and do it as it's own thing in a smaller PR =)